NebulaHost/PacketProcessors/Logistics/StationUIBroadcaster.cs

```diff
@@ -23,27 +23,33 @@ public StationUIBroadcaster()
         public void ProcessPacket(StationUI packet, NebulaConnection conn)
         {
             // if a user adds/removes a ship, drone or warper or changes max power input broadcast to everyone.
-            if((packet.settingIndex == StationUI.UIsettings.MaxChargePower || packet.settingIndex == StationUI.UIsettings.setDroneCount || packet.settingIndex == StationUI.UIsettings.setShipCount || packet.settingIndex == StationUI.UIsettings.setWarperCount) && StationUIManager.UpdateCooldown == 0)
+            if (StationUIManager.UpdateCooldown == 0 &&
+                (packet.SettingIndex == StationUI.EUISettings.MaxChargePower
+                 || packet.SettingIndex == StationUI.EUISettings.SetDroneCount
+                 || packet.SettingIndex == StationUI.EUISettings.SetShipCount
+                 || packet.SettingIndex == StationUI.EUISettings.SetWarperCount
+                 )
+                )
             {
+                // this is the SendPacketToAllPlayers() logic but we need to set the mimic flag here.
                 using (playerManager.GetConnectedPlayers(out var connectedPlayers))
                 {
-                    // this is the SendPacketToAllPlayers() logic but we need to set the mimic flag here.
                     foreach (var kvp in connectedPlayers)
                     {
                         Player p = kvp.Value;
-                        if(p.Connection == conn)
+                        if (p.Connection == conn)
                         {
-                            packet.shouldMimick = true;
+                            packet.ShouldMimic = true;
                         }
                         p.SendPacket(packet);
                     }
                 }
             }
-            else if(packet.settingIndex == StationUI.UIsettings.addOrRemoveItemFromStorageResp)
+            else if (packet.SettingIndex == StationUI.EUISettings.AddOrRemoveItemFromStorageResponse)
```

Comment:
Check for item changes here and not in the if() above because we only need to sync item changes to players on the same planet.